app/routes/languages.rb

```diff
@@ -70,6 +70,11 @@ def topic_not_found(topic)
         status 404
         erb :"languages/not_found", locals: { track_id: topic }
       end
+
+      def active(topic, selected_topic)
+        'active' if topic == selected_topic
+      end
```

Comment:
What's the difference between a method definition and a conditional?
Why is one appropriate and the other not?

(Ideally I'd have an `active_class` method on the track object and there would be no code in the template at all - as it should be.)